presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -179,6 +184,7 @@ class TaskManager {
 
   std::string baseUri_;
   std::string nodeId_;
+  std::string baseSpillDirectory_;
```

Comment:
nit: s/baseSpillDirectory_/baseSpillDir_/

Reply:
Addressing in https://github.com/prestodb/presto/pull/20582